### PR TITLE
Better path parameter names for organization and member APIs

### LINKED ISSUE #35745: GET .../organizations/{id}/members/{id} multiple ids
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

organizations

### Describe the bug

These two endpoints
- `/admin/realms/{realm}/organizations/{id}/members/{id}`
- `/admin/realms/{realm}/organizations/{id}/members/{id}/organizations`

use `{id}` twice which prevents generated HTTP client code from being able to use it correctly without manual modification of the OpenAPI JSON spec file. Suggest renaming to `{orgId}` and `{memberId}` or something along those lines.

image

### Version

26.0.7

### Regression

- [ ] The issue is a regression

### Expected behavior

Path variable names are distinct.

### Actual behavior

Path variable names are NOT distinct.

### How to Reproduce?

- View the OpenAPI JSON spec file
- Or generate an HTTP client using the spec file (with e.g. NSwag) and see that you cannot pass distinct values for organization ID and member ID

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by pedroigor**
Closes #35745

### CODE REVIEW

**Review by sguilhen — APPROVED** [review]

File: services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java [hunk 1]
```diff
@@ -154,29 +154,29 @@ public Stream<MemberRepresentation> search(
...
+    @Path("{memberId}")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @NoCache
     @Tag(name = KeycloakOpenAPI.Admin.Tags.ORGANIZATIONS)
     @Operation( summary = "Returns the member of the organization with the specified id", description = "Searches for a" +
             "user with the given id. If one is found, and is currently a member of the organization, returns it. Otherwise," +
             "an error response with status NOT_FOUND is returned")
-    public MemberRepresentation get(@PathParam("id") String id) {
+    public MemberRepresentation get(@PathParam("memberId") String id) {
```
Comment (ahus1) [inline services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java:165]: Nitpick:

```suggestion
    public MemberRepresentation get(@PathParam("memberId") String memberId) {
```

**Review by ahus1 — DISMISSED** [review]
Approving based on @sguilhen's review. Still one nit-pick below to keep things consistent. Leaving it to you if you want to merge it as is, or rework.

File: services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java (hunk 1 above)
Comment (pedroigor) [reply services/src/main/java/org/keycloak/organization/admin/resource/OrganizationMemberResource.java]: Thanks. I did your suggestion and also updated the PR to use `member-id` as the path param. Noticed we are already using something like `user-id` in the User API so I updated it to follow the same format.

**Review by sguilhen — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]
Approving based on @sguilhen's review.

**Pull request merged**